drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-client/src/main/java/org/drools/workbench/screens/dtablexls/client/editor/DecisionTableXLSEditorPresenter.java

```diff
@@ -123,6 +123,7 @@ public void onUpload() {
 
     void submit() {
         view.submit(versionRecordManager.getCurrentPath());
+        versionRecordManager.restoreToCurrentVersion();
```

Comment:
Are you sure this is the correct method!?

From what I can see (from the code) that will show a popup to restore the file to the current version.

Other editors use ```versionRecordManager.reloadVersions(path);``` in their "save success callbacks".

You might also want to check the XLS Score Card Editor and the "Upload file" Editor for similar problems.

I'll compile this PR and _see_ what it's doing; in case my code review is misguided.

Reply:
Ah, right, sorry. Weekend is too close to do some coding.